### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by MishaLehotskyi**
[DEMO LINK](https://MishaLehotskyi.github.io/react_dynamic-list-of-todos/)

### CODE REVIEW

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Nice start! Let's fix and improve your work a little bit)

File: src/components/TodoList/TodoList.tsx [hunk 1]
```diff
@@ -1,100 +1,74 @@
...
-          </button>
-        </td>
-      </tr>
-    </tbody>
-  </table>
-);
+      <tbody>
+        {todos.map(todo => {
+          return (
+            <tr data-cy="todo" className="">
```
Comment (anastasiiavorobiova) [inline src/components/TodoList/TodoList.tsx:34]: Consider using the key attribute here

Comment (anastasiiavorobiova) [inline src/components/TodoList/TodoList.tsx:34]:
```suggestion
            <tr data-cy="todo">
```

File: src/components/TodoList/TodoList.tsx [hunk 2]
```diff
@@ -1,100 +1,74 @@
...
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-    </tbody>
-  </table>
-);
+      <tbody>
+        {todos.map(todo => {
```
Comment (anastasiiavorobiova) [inline src/components/TodoList/TodoList.tsx:32]: Consider using destructuring here

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Added a few comments to fix

**Review by Esceype — APPROVED** [review]
Well done!!!

**Pull request closed without merging**